frontend/src/components/Register.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react';
+import APIService from '../components/APIService';
+import { useCookies } from 'react-cookie';
+import { useNavigate } from 'react-router-dom';
+
+function Register() {
+  const [name, setName] = useState('');
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [re_password, setRe_password] = useState('');
+  const [email, setEmail] = useState('');
```

Comment:
instead of using separate `useState` for each input value you can do something like that

```
  const [userConfiguration, setUserConfiguration] = useState({
    name: "",
    email: "",
    password: ""
  });

  const handleChange = (event) =>
    setUserConfiguration({
      ...userConfiguration,
      [event.target.name]: event.target.value
    });

  return (
    <div>
      <input name="name" value={userConfiguration.name} onChange={handleChange} />
      <input name="email" value={userConfiguration.email} onChange={handleChange} />
      <input name="password" value={userConfiguration.password} onChange={handleChange} />
    </div>
  );
```

Reply:
of course set proper names for each variable